plugins/utils/asyncrun.nix

```diff
@@ -0,0 +1,9 @@
+{ lib, pkgs, config, ... }@attrs:
```

Comment:
Couldn't this go in ./generated.nix?

Reply:
Yes. I made this before coming up with the idea for `generated.nix`